docs/src/content/docs/de/components/using-components.mdx

```diff
@@ -0,0 +1,96 @@
+
+```mdx
+---
+# src/content/docs/example.mdx
+title: Willkommen bei meiner Dokumentation
+---
+
+import { Icon } from '@astrojs/starlight/components';
+import CustomCard from '../../components/CustomCard.astro';
+
+<Icon name="open-book" />
+
+<CustomCard>
+	Komponenten können auch **verschachtelte Inhalte** enthalten.
+</CustomCard>
+```
+
+Da Starlight von Astro unterstützt wird, kannst du in deinen MDX-Dateien Unterstützung für Komponenten hinzufügen, die mit einem beliebigen [unterstützten UI-Framework (React, Preact, Svelte, Vue, Solid und Alpine)](https://docs.astro.build/de/core-concepts/framework-components/) erstellt wurden.
+Weitere Informationen über [Verwendung von Komponenten in MDX](https://docs.astro.build/en/guides/integrations-guide/mdx/#using-components-in-mdx) findest du in der Astro-Dokumentation.
+
+## Verwendung einer Komponente in Markdoc
+
+Füge Unterstützung für das Verfassen von Inhalten in Markdoc hinzu, indem du unsere [Markdoc-Einrichtungsanleitung](/de/guides/authoring-content/#markdoc) befolgst.
+
+Mit der Starlight Markdoc-Voreinstellung kannst du die in Starlight eingebauten Komponenten mit der Markdoc-Tag-Syntax `{% %}` verwenden.
+Im Gegensatz zu MDX müssen die Komponenten in Markdoc nicht importiert werden.
+Das folgende Beispiel stellt Starlights [Kartenkomponente](/de/components/cards/) in einer Markdoc-Datei dar:
+
+```markdoc
+---
+# src/content/docs/example.mdoc
+title: Willkommen bei meiner Dokumentation
+---
+
+{% card title="Stars" icon="star" %}
+Sirius, Wega, Betelgeuse
+{% /card %}
+```
+
+Weitere Informationen zur Verwendung von Komponenten in Markdoc-Dateien findst du in der [Astro Markdoc Integrationsdokumentation](https://docs.astro.build/de/guides/integrations-guide/markdoc/#komponenten-rendern).
+
+## Integrierte Komponenten
+
+Starlight bietet eingebaute Komponenten für gängige Dokumentationsanwendungen.
+Diese Komponenten sind aus dem Paket `@astrojs/starlight/components` in MDX-Dateien und aus dem [Starlight Markdoc preset](/de/guides/authoring-content/#markdoc) in Markdoc-Dateien verfügbar.
+
+In der Seitenleiste findst du eine Liste der verfügbaren Komponenten und deren Verwendung.
+
+## Kompatibilität mit den Stilen von Starlight
+
+Starlight wendet Standardstile auf deinen Markdown-Inhalt an, z.B. fügt es einen Rand zwischen den Elementen hinzu.
+Wenn diese Stile mit dem Erscheinungsbild deiner Komponente in Konflikt stehen, setze die Klasse `not-content` für deine Komponente, um sie zu deaktivieren.
+
+```astro 'class="not-content"'
+---
+// src/components/Example.astro
+---
+
+<div class="not-content">
+	<p>Die Standard-Inhaltsgestaltung von Starlight wirkt sich nicht aus.</p>
+</div>
+```
+
+## Komponenteneigenschaften
+
+Verwende den Typ [`ComponentProps`](https://docs.astro.build/de/guides/typescript/#componentprops-typ) aus `astro/types`, um auf die von einer Komponente akzeptierten `Props` zu verweisen, auch wenn sie nicht von der Komponente selbst exportiert werden.
```

Comment:
```suggestion
Verwende den Typ [`ComponentProps`](https://docs.astro.build/de/guides/typescript/#componentprops-type) aus `astro/types`, um auf die von einer Komponente akzeptierten `Props` zu verweisen, auch wenn sie nicht von der Komponente selbst exportiert werden.
```

This section does not exist in the old version of the German page so let's keep the English anchor for now.